web/src/components/molecules/Common/Form/AssetItem/index.tsx

```diff
@@ -141,4 +154,27 @@ const AssetButton = styled(Button)`
   border: 1px dashed #d9d9d9;
 `;
 
+const AssetWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const AssetIcon = styled(Button)`
+  color: #000000d9;
+  margin-top: 4px;
+  top: 3px;
+`;
+
+const AssetName = styled(Button)`
+  color: #1890ff;
+  margin-left: 12px;
+`;
```

Comment:
AssetIcon and AssetName don't really convey what they should. I would assume they are an Icon and then Text. Not two buttons. Please rename them